packages/i18n/src/messages/de.json

```
@@ -2674,15 +2679,38 @@
     "callControlsScreenshare": "Bildschirmfreigabe",
     "callControlsScreenshareActiveAria": "Bildschirmfreigabe, aktiv",
     "callControlsScreenshareInactiveAria": "Bildschirmfreigabe, inaktiv",
+    "callReactButton": "Reagieren",
```

Biome (2.4.16)

[error] The key callReactButton was already declared.

(lint/suspicious/noDuplicateObjectKeys)